server/src/main/java/org/elasticsearch/action/admin/cluster/repositories/cleanup/TransportCleanupRepositoryAction.java

```diff
@@ -122,7 +122,7 @@ private static ClusterState removeInProgressCleanup(final ClusterState currentSt
         RepositoryCleanupInProgress cleanupInProgress = currentState.custom(RepositoryCleanupInProgress.TYPE);
         if (cleanupInProgress != null) {
             boolean changed = false;
-            if (cleanupInProgress.cleanupInProgress() == false) {
+            if (cleanupInProgress.cleanupInProgress()) {
```

Comment:
I was confused with `cleanupInProgress()` returning `entries.isEmpty()`, so we talked via another channel and agreed to rename this method `hasCleanupsInProgress()` as @original-brownbear suggested.